packages/react-devtools-shared/src/devtools/views/Components/Tooltip.js

```diff
@@ -0,0 +1,51 @@
+/** @flow */
+
+import React, {useCallback, useRef, useState} from 'react';
+import {useSmartTooltip} from '../hooks';
+
+import styles from './Tooltip.css';
+
+const initialTooltipState = {height: 0, mouseX: 0, mouseY: 0, width: 0};
+
+export default function Tooltip({children, label}: any) {
+  const containerRef = useRef(null);
+  const [tooltipState, setTooltipState] = useState(initialTooltipState);
+  const tooltipRef = useSmartTooltip(tooltipState);
+
+  const onMouseMove = useCallback(
```

Comment:
`useCallback` doesn't do anything in this case, so we can remove it :)

Reply:
Done